components/browser/state/src/main/java/mozilla/components/browser/state/state/CustomTabSessionState.kt

```diff
@@ -29,7 +30,8 @@ data class CustomTabSessionState(
     override val extensionState: Map<String, WebExtensionState> = emptyMap(),
     override val mediaSessionState: MediaSessionState? = null,
     override val contextId: String? = null,
-    override val source: SessionState.Source = SessionState.Source.CUSTOM_TAB
+    override val source: SessionState.Source = SessionState.Source.Internal.CustomTab,
+    override val restored: Boolean = false
```

Comment:
In theory we could add `restored` only to `TabSessionState`, but we may deal with `SessionState` instances somewhere?

Reply:
I've went that route at first, and I ran into once instance of `SessionState` (in Fenix's back button handling).